src/server/apps/worldserver/worldserver.conf.dist

```diff
@@ -1307,6 +1307,14 @@ DeletedCharacterTicketTrace = 0
 
 DungeonFinder.OptionsMask = 5
 
+#
+#   DBC.EnforceItemAttributes
+#        Disallow overriding item attributes stored in DBC files with values from the database
+#        Default: 0 - Off, Use DB values
+#                 1 - On, Enforce DBC Values (default)
+
+DBC.EnforceItemAttributes = 1
```

Comment:
Is this config really necessary? AC already overrides all other DBC tables from DB values with no config option for specific tables, what is it about item.DBC that necessitates this?

Reply:
> 

the conf gives us the option to enforce the dbc value or not and to use the db value.
at the moment we do not using the dbc at all as evident with me adding it in the dbcstore. 
there is no over ride table for item_dbc at all currently as seen in the dbcstore.
we have alot of hard coded defines values being used and not using the dbc at all as evident in the objectmgr.cpp.
why we are not using the item.dbc at all is beyond me.